controllers/replicationdestination_controller.go

```diff
@@ -334,6 +341,13 @@ func (r *rsyncDestReconciler) ensureJob(l logr.Logger) (bool, error) {
 		return false, err
 	}
 
+	// If PauseSync has been issued but Parallelism is not 0 delete job
+	if r.Instance.Spec.Rsync.PauseSync && *r.job.Spec.Parallelism != int32(0) {
```

Comment:
I don't know if `parallelism` is mutable...
If it is mutable, this shouldn't be necessary.
If not, you're going to run into problems w/ `CreateOrUpdate`, above. You'll need to put the parallelism stuff inside a `if r.job.CreationTimestamp.IsZero() {` check so you only try to set it on 1st create. (See PVC handling code in `volumehandler.go`)

Also, this handles the case where you "set" pause. What about "unsetting" it? (pause==false; parallelism==0)